### DISCUSSION TITLE
Round or smooth the boundary edges

### DISCUSSION THREAD

**Discussion by 1939938853** [pyvista/pyvista#5008] [Q&A]
Hello,

Just wonder if it is possible that pyvista can round or smooth the boundary edges like this and how.

Thank you

![image](https://github.com/pyvista/pyvista/assets/59957922/46520ee9-8aae-4346-8120-bdccd8fd7a82)

**Comment by banesullivan** [comment]
A place to start might be to try https://docs.pyvista.org/version/stable/api/core/_autosummary/pyvista.PolyDataFilters.smooth.html#pyvista.PolyDataFilters.smooth

Otherwise this may be tough to solve with PyVista/VTK alone
